settings.gradle.kts

```diff
@@ -26,6 +26,9 @@ include(
     ":kmath-core",
     ":kmath-coroutines",
     ":kmath-functions",
+    ":test-utils-functions",
```

Comment:
Are those tests used more than in one module? If not, please move them to appropriate module test source sets instead of separate modules.

Reply:
Done.